globe/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />
+  <script src='https://unpkg.com/maplibre-gl@5.0.0-pre.1/dist/maplibre-gl.js'></script>
```

Comment:
I would recommend waiting for an official release...

Reply:
I think we should have some maplibre-hosted demo for the newsletter. Would you prefer to have it e.g. in the maplibre website instead @HarelM ?

Reply:
The demo can wait for an official release to be made, no need to rush it.
It's only the first pre-release.